### PR TITLE
[ltx] Update to v3, [@xmpp/xml] Update to v0.13

### PR DESCRIPTION
**Pull request by BendingBender**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/xmppjs/ltx/releases/tag/v3.0.0
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Comment by sandersn** [comment]
How hard would it be to update xmpp__xml to ltx's new types? `@xmpp/xml`'s latest version depends on ltx@3, so I think it makes sense. And ltx mentions jabber in its comments, making me think the two projects are related.

**Comment by BendingBender** [comment]
hm, no idea, haven't looked into it

**Comment by BendingBender** [comment]
@sandersn Updated `@xmpp/xml` along with `ltx`, should I close the issue in DTS-tools?

File: types/ltx/package.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "private": true,
+    "exports": {
+        ".": {
+            "types": {
+                "import": "./src/ltx.d.ts",
```
Comment (weswigham) [inline types/ltx/package.json:6]: Both of these entrypoints will be cjs format, afaik, which... works, I guess? I just don't get why.

Comment (BendingBender) [reply types/ltx/package.json:6]: I thought, the first will be mjs. But if you say so, I'll remove the package.json.

Comment (weswigham) [reply types/ltx/package.json]: It'll be the file looked up by an `import`, yes, but it'll only be a `mjs` format file if the file itself has a `.d.mts` extension.

Comment (weswigham) [reply types/ltx/package.json]: It's definitely correct to remove it though - generally speaking, DT packages only need an export map if the source package has one, and it seems like ltx [does not](https://github.com/xmppjs/ltx/blob/main/package.json).

Comment (weswigham) [reply types/ltx/package.json]: Ack, but it does use `"type": "module",` and `"type": "commonjs",` for the `lib` folder - afaik, we'd need to duplicate that - add `"type": "module",` to this package json and add a package json in the lib folder with `"type": "commonjs"`, and we can make this present the same formats. I... do not know if the DT automation is up to the task of allowing a nested `package.json`.

Comment (weswigham) [reply types/ltx/package.json]: Maybe it just works if it's listed in `OTHER_FILES.txt`? I dunno.

Comment (BendingBender) [reply types/ltx/package.json]: Let's try it out.

Comment (BendingBender) [reply types/ltx/package.json]: OK, the build didn't fail but no idea whether it will be published correctly.

**Review by weswigham — APPROVED** [review]
Only one thing to do, I guess - try to publish it and see what actually ends up in the package (and fix DT tools if it doesn't work)

**Comment by BendingBender** [comment]
Ready to merge

**Comment by BendingBender** [comment]
@weswigham Doesn't look like it has worked correctly... There is no `lib/package.json` in the published package.

**Pull request merged**